pkg/e2e/e2e.go

```diff
@@ -102,84 +101,73 @@ func runGinkgoTests() error {
 		log.Printf("Could not create reporter directory: %v", err)
 	}
 
-	if !cfg.DryRun {
```

Comment:
Yes. Everything still gets printed out, but now it will also print out the tests that would have been run rather than stopping before invoking Ginkgo.